cfme/tests/v2v/test_v2v_migrations_single_vcenter.py

```diff
@@ -0,0 +1,376 @@
+         not request_details_list.is_errored(vm))
+
+
+@pytest.mark.parametrize('form_data_vm_obj_single_datastore', [['nfs', 'nfs', rhel7_minimal]],
+                        indirect=True)
+def test_migration_special_char_name(request, appliance, v2v_providers, host_creds, conversion_tags,
+                                    form_data_vm_obj_single_datastore):
+    """Tests migration where name of migration plan is comprised of special non-alphanumeric
+       characters, such as '@#$(&#@('.
+
+    Polarion:
+        assignee: kkulkarn
+        casecomponent: V2V
+        initialEstimate: None
+        subcomponent: RHV
+        upstream: yes
+    """
+    infrastructure_mapping_collection = appliance.collections.v2v_mappings
+    mapping = infrastructure_mapping_collection.create(form_data_vm_obj_single_datastore.form_data)
+
+    @request.addfinalizer
+    def _cleanup():
+        infrastructure_mapping_collection.delete(mapping)
+
+    migration_plan_collection = appliance.collections.v2v_plans
+    # fauxfactory.gen_special() used here to create special character string e.g. #$@#@
+    migration_plan = migration_plan_collection.create(
+        name="{}".format(fauxfactory.gen_special()), description="desc_{}"
+        .format(fauxfactory.gen_alphanumeric()), infra_map=mapping.name,
+        vm_list=form_data_vm_obj_single_datastore.vm_list, start_migration=True)
+
+    # explicit wait for spinner of in-progress status card
+    view = appliance.browser.create_view(
+        navigator.get_class(migration_plan_collection, 'All').VIEW.pick())
+    wait_for(func=view.progress_card.is_plan_started, func_args=[migration_plan.name],
+        message="migration plan is starting, be patient please", delay=5, num_sec=150,
+        handle_exception=True, fail_cond=False)
+
+    # wait until plan is in progress
+    wait_for(func=view.plan_in_progress, func_args=[migration_plan.name],
+        message="migration plan is in progress, be patient please",
+        delay=5, num_sec=1800)
+
+    view.switch_to("Completed Plans")
+    view.wait_displayed()
+    migration_plan_collection.find_completed_plan(migration_plan)
+    logger.info("For plan %s, migration status after completion: %s, total time elapsed: %s",
+        migration_plan.name, view.migration_plans_completed_list.get_vm_count_in_plan(
+            migration_plan.name), view.migration_plans_completed_list.get_clock(
+            migration_plan.name))
+    # validate MAC address matches between source and target VMs
+    assert view.migration_plans_completed_list.is_plan_succeeded(migration_plan.name)
+    src_vm = form_data_vm_obj_single_datastore.vm_list[0]
+    migrated_vm = get_migrated_vm_obj(src_vm, v2v_providers.rhv_provider)
+    assert src_vm.mac_address == migrated_vm.mac_address
+
+
+def test_migration_long_name(request, appliance, v2v_providers, host_creds, conversion_tags):
+    """Test to check VM name with 64 character should work
+
+    Polarion:
+        assignee: None
```

Comment:
Need assignee here as well as casecomponent

Reply:
casecomponent isn't required yet, I'm updating that in separate PR with bulk setting of all the currently missing ones.